sdk/src/main/java/com/horizen/account/event/Anonymous.java

```diff
@@ -0,0 +1,10 @@
+package com.horizen.account.event;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE})
+public @interface Anonymous {}
```

Comment:
Put `Anonymous`, `Indexed` and `Parameter` to `account.event.annotation` package